.travis.yml

```diff
@@ -22,7 +22,7 @@ env:
   - VJS=5
   - VJS=6
 addons:
-  firefox: latest
+  firefox: "54.0"
```

Comment:
We should add a comment with the Travis issue and a TODO, because this is fragile for users, since most Firefox users will be on a more modern version (meaning out tests should use more modern versions).